ts/Extensions/Annotations/NavigationBindings.ts

```diff
@@ -228,6 +235,35 @@ const bindingsUtils = {
         } as Record<string, ('checkbox'|'number'|'text')>)[
             typeof value
         ];
+    },
+    /**
+     * Returns the first xAxis or yAxis that was clicked with its value.
+     *
+     * @private
+     * @function Highcharts.NavigationBindingsUtilsObject#getAssignedAxis
+     *
+     * @param {Array<Highcharts.PointerAxisCoordinateObject>} coords
+     *        All the chart's x or y axes with a current pointer's axis value.
+     *
+     * @return {Highcharts.PointerAxisCoordinateObject}
+     *         Object with a first found axis and its value that pointer
+     *         is currently pointing.
+     */
+    getAssignedAxis(
+        coords: Array<Highcharts.PointerAxisCoordinateObject>
+    ): Highcharts.PointerAxisCoordinateObject {
+        return coords.filter(function (coord): boolean {
+            const axisMin = coord.axis.min as number,
+                axisMax = coord.axis.max as number,
```

Comment:
In this case, as the previous comment, we cannot cast `axis.min` and `axis.max` to `number`, because they aren't necessarily numbers. Instead we need to code for the case that they are not numbers.

In practice, we can test this by setting `data: []` in the given unit test.